test/unit/io.js

```diff
@@ -861,6 +861,70 @@ describe('Input/output', function () {
     });
   });
 
+  it('TIFF lzw compression shrinks test file', function (done) {
+    const startSize = fs.statSync(fixtures.inputUncompressedTiff).size;
```

Comment:
Thank you. I hope you understand that PRs where tests are failing cannot be merged and code that is untested cannot be safely maintained, refactored or otherwise improved.